packages/x-date-pickers/src/DayCalendarSkeleton/DayCalendarSkeleton.tsx

```diff
@@ -92,22 +88,22 @@ function DayCalendarSkeleton(inProps: DayCalendarSkeletonProps) {
     name: 'MuiDayCalendarSkeleton',
   });
 
-  const { className, ...other } = props;
+  const { className, classes: classesProp, ...other } = props;
 
-  const classes = useUtilityClasses(other);
+  const classes = useUtilityClasses(classesProp);
 
   return (
     <DayCalendarSkeletonRoot className={clsx(classes.root, className)} {...other}>
       {monthMap.map((week, index) => (
         <DayCalendarSkeletonWeek key={index} className={classes.week}>
-          {week.map((day, index2) => (
+          {week.map((dayInMonth, index2) => (
             <DayCalendarSkeletonDay
               key={index2}
               variant="circular"
               width={DAY_SIZE}
               height={DAY_SIZE}
               className={classes.daySkeleton}
-              ownerState={{ day }}
+              data-day-in-month={dayInMonth}
```

Comment:
I can create a `DayCalendarSkeletonOwnerState` but the component is probably not used a lot and even less overriden with `styleOverrides` so I went with simplicity here